src/main/java/com/blog/web/dto/ArticleDto.java

```diff
@@ -0,0 +1,117 @@
+package com.blog.web.dto;
+
+import com.blog.web.models.Article;
+import com.blog.web.models.UserEntity;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotEmpty;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.validator.constraints.URL;
+
+import java.time.LocalDateTime;
+
+public class ArticleDto {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
why do you need non-primative... also call the variables final if you don't intend for mutability